plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -221,8 +280,16 @@ fu_plugin_thunderbolt_add (FuPlugin *plugin, GUdevDevice *device)
 		}
 	}
 	if (!is_safemode) {
+		if (is_host)
+			if (!fu_plugin_thunderbolt_is_native (device, &is_native, &error)) {
+				g_warning ("failed to get native mode status: %s", error->message);
```

Comment:
The other prints above are `g_warning`, but maybe this one should be `g_error`?

Reply:
I agree this should be g_error.

Reply:
Fixed